core/ui/SideBar/Open.tid

```diff
@@ -23,10 +23,20 @@ $button$
 </$droppable>
 \end
 
+\define open-tab-close-tiddler-button()
+<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class="tc-btn-invisible tc-btn-mini tc-small-gap-right">{{$:/core/images/close-button}}</$button>
+\end
+
+<!-- Override one or both of the following 2 macros with a global macro -->
+<!-- of the same name if you need to change how elements are displayed  -->
+\define open-tab-link()<$link to={{!!title}}><$view field="title"/></$link>
+
+\define open-tab-element()<<open-tab-close-tiddler-button>><<open-tab-link>>
+
 <div class="tc-sidebar-tab-open">
 <$list filter="[list<tv-story-list>]" history=<<tv-history-list>> storyview="pop">
 <div class="tc-sidebar-tab-open-item">
-<$macrocall $name="droppable-item" button="""<$button message="tm-close-tiddler" tooltip={{$:/language/Buttons/Close/Hint}} aria-label={{$:/language/Buttons/Close/Caption}} class="tc-btn-invisible tc-btn-mini tc-small-gap-right">{{$:/core/images/close-button}}</$button><$link to={{!!title}}><$view field="title"/></$link>"""/>
+<$macrocall $name="droppable-item" button="<<open-tab-element>>"/>
```

Comment:
I think the words "list-item" might be better than "element" here

Reply:
The problem is, that we have to have global macros. So I would like to use a name, that works for the core, but doesn't clash if a user has a similar one. IMO `list-item` may be easy to clash with future user names.